docs/advanced-features/compiler.md

```diff
@@ -30,19 +30,7 @@ We chose to build on SWC for a few reasons:
 - **WebAssembly:** Rust's support for WASM is essential for supporting all possible platforms and taking Next.js development everywhere.
```

Comment:
The version table at the top of this doc should mention which version it went stable and any other changes

Reply:
I'm going to keep this PR to just the new transforms for now. I've asked @timneutkens about Jest and wasm, so I can address those in a follow up.